source/MaterialXRuntime/Private/PrvElement.h

```diff
@@ -140,15 +149,13 @@ class PrvElement : public PrvObject
     static const string PATH_SEPARATOR;
 
 protected:
-    PrvElement(RtObjType objType, PrvElement* parent, const RtToken& name);
```

Comment:
Removed setting parent in element constructors. Instead a child's parent is set in addChild, when the actual child-parent relationship is created.